ui/imports/shared/status/StatusETHTransactionModal.qml

```diff
@@ -112,6 +112,7 @@ ModalPopup {
                 visible: false
                 accounts: root.ensUsernamesStore.walletAccounts
                 contactsStore: root.contactsStore
+                isValid: true
```

Comment:
@anastasiyaig I would say that this is not something you should set from outside. Also in one point this should be private prop of `RecipientSelector` component. So far I would just remove it.

Reply:
@richard-ramos i squashed commits and your name disappeared from this branch , but this is related to the changes u implemented, please take a look

Reply:
@saledjenic I set the `isValid` option because changes done recently in https://github.com/status-im/status-desktop/blob/master/ui/imports/shared/controls/RecipientSelector.qml broke this modal and made it difficult to use a `RecipientSelector` with a default `selectedRecipient` and `visible: false`.  (cc: @faust4exe for ideas on a possible solution to this scenario )

I'll try getting rid completely of the `RecipientSelector` in this modal, as this UI is meant to be used when interacting with smart contracts, and the `to:` of the eth transaction is not meant to be changed by the user.

Reply:
Can confirm that it works :)